sway-ir/tests/inline/get_storage_key.ir

```diff
@@ -4,6 +4,12 @@
 
 script {
     fn anon_0() -> b256 {
```

Comment:
I can kind of tell what is going on here but the function name could be improved. Similar to the other `anon` functions

Reply:
So the IR generator chooses those, but you're absolutely right: there is no need for a test to also have non-descriptive names like that.